packages/phone-number-privacy/combiner/src/pnp/services/account-services.ts

```diff
@@ -34,7 +35,8 @@ export class CachingAccountService implements AccountService {
       const dek = await this.cache.fetch(address)
 
       if (dek === undefined) {
-        // TODO decide which error ot use here
+        Counters.errors.labels('NA', ErrorMessage.FULL_NODE_ERROR).inc()
+        Counters.blockchainErrors.labels('NA', ErrorMessage.FAILURE_TO_GET_DEK).inc()
```

Comment:
This error might be misleading, as this is reading from cache, not blockchain

Perhaps it would make sense to add a cache error counter